kingpin/actors/aws/ecs.py

```diff
@@ -32,7 +32,7 @@
 
 log = logging.getLogger(__name__)
 
-__author__ = 'Steve Mostovoy <smostovoy@nextdoor.com>'
```

Comment:
We might as well have exceptions and stack traces routed to a current Nextdoorian...